Simplenote/en.lproj/Localizable.strings

```diff
@@ -635,3 +662,6 @@
 /* Password Requirements: Title */
 "Your password is insecure and must be reset. The password requirements are:" = "Your password is insecure and must be reset. The password requirements are:";
 
+/* Indicates that Extended Debugging has been enabled */
+"🐞 Debug Enabled 🐞" = "🐞 Debug Enabled 🐞";
```

Comment:
This reminds me of a debugging extension I use to use in the browser. It had a blue ladybird as the logo.